src/server/routes/methods.js

```diff
@@ -41,5 +41,41 @@ export default [
       notes: 'Retrieve all the fishing methods in the database',
       tags: ['api', 'methods']
     }
+  },
+  {
+    method: 'GET',
+    path: '/methods/{methodId}',
+    options: {
+      /**
+       * Retreive a fishing method by ID from the database
+       *
+       * @param {import('@hapi/hapi').Request request - The Hapi request object
+       * @param {import('@hapi/hapi').ResponseToolkit} h - The Hapi response toolkit
+       * @returns {Promise<import('@hapi/hapi').ResponseObject>} - A response containing the target {@link Method}
+       */
+      handler: async (request, h) => {
+        try {
+          const methodId = request.params.methodId
+          const method = await Method.findOne({ where: { id: methodId } })
+
+          if (!method) {
+            logger.error(`method not found for id ${methodId}`)
+            return h.response().code(StatusCodes.NOT_FOUND)
+          }
+
+          const mappedMethod = mapMethodToResponse(request, method.toJSON())
+
+          return h.response(mappedMethod).code(StatusCodes.OK)
+        } catch (error) {
+          logger.error('Error fetching method:', error)
+          return h
+            .response({ error: 'Unable to fetch method' })
+            .code(StatusCodes.INTERNAL_SERVER_ERROR)
+        }
```

Comment:
This and the not found above feel like candidates to use the new handleNotFound / handleServerError methods